scripts/build-homepage

```diff
@@ -3,10 +3,11 @@
 # build the applications
 mkdir -p build
 cd build
-cmake -DPLUGINS='ALL;-fstab;-semlock' -DBUILD_FULL=OFF -DBUILD_SHARED=ON -DBUILD_STATIC=OFF -DBUILD_DOCUMENTATION=OFF -DTOOLS='kdb;rest-backend;rest-frontend' ..
+cmake -DPLUGINS='ALL;-fstab;-semlock' -DBUILD_FULL=OFF -DBUILD_SHARED=ON -DBUILD_STATIC=OFF -DBUILD_DOCUMENTATION=OFF -DTOOLS='kdb;rest-backend;rest-frontend' -DCMAKE_COMPILE_FLAGS='-D_FORTIFY_SOURCE=2 -Wformat-security -fstack-protector-strong -Wstack-protector' -DCMAKE_EXE_LINK_FLAGS='-Wl,-z,now' ..
```

Comment:
-Wformat-security is standard anyway, -Werror=format-security would be interesting.

And ENABLE_ASAN is missing, or is it too slow? Did you try with httperf?